plutus-core/plutus-core/src/PlutusCore/Pretty/Readable.hs

```diff
@@ -100,6 +100,28 @@ deriving via
   instance
     (PrettyDefaultBy config (AsReadable a)) => PrettyBy config (AsReadable a)
 
+newtype AsReadableWithUniques a = AsReadableWithUniques
+  { unAsReadableWithUniques :: a
+  }
```

Comment:
`AsDefaultWithUniques` would be useful too... I wonder if we could keep `AsReadable` and `AsWithUniques` separate and define `AsDefaultWithUniques` in terms of that. Or have `As '[Readable, WithUniques]` or something.